packages/blade/src/components/Input/TextArea/TextArea.tsx

```diff
@@ -115,12 +118,33 @@ const _TextArea: React.ForwardRefRenderFunction<BladeElementRef, TextAreaProps>
     numberOfLines = 2,
     testID,
     size = 'medium',
+    isTaggedInput,
+    tags,
+    onTagChange,
     ...styledProps
   },
   ref,
 ) => {
   const inputRef = React.useRef<BladeElementRef>(null);
   const mergedRef = useMergeRefs(ref, inputRef);
+  const [isInputFocussed, setIsInputFocussed] = React.useState(autoFocus ?? false);
+  const {
+    activeTagIndex,
+    setActiveTagIndex,
+    getTags,
+    handleTaggedInputKeydown,
+    handleTaggedInputChange,
+    handleTagsClear,
+  } = useTaggedInput({
+    tags,
+    onTagChange,
+    isDisabled,
+    inputRef: inputRef as React.RefObject<BladeElementRefWithValue>,
```

Comment:
Just define the inputRef type as `React.RefObject<BladeElementRefWithValue>` in the useTaggedInput type only